nest-forms-backend/src/main.ts

```diff
@@ -29,7 +29,8 @@ async function bootstrap(): Promise<void> {
     methods: 'GET,HEAD,PUT,PATCH,POST,DELETE,OPTIONS',
     preflightContinue: false,
     credentials: true,
-    allowedHeaders: 'Content-Type, Accept, Authorization',
+    allowedHeaders:
+      'Content-Type, Accept, Authorization, X-Cognito-Guest-Identity-Id',
```

Comment:
Only change for now in BE is to accept the header, as BE is not deployed in cypress test they will fail because of this (server will reject it).

Reply:
OK, all requests that are in E2E tests are ran while authenticated.